README.md

```diff
@@ -72,3 +64,20 @@ If you would like to contribute to Dialtone Vue the first step is to get the pro
 4. Visit local dev server at http://localhost:9011/
 
 Next read the more detailed contributor documentation in [CONTRIBUTING.md](.github/CONTRIBUTING.md).
+
+## Project Status
+
+Dialtone Vue is a new project, and as such it is under constant development as we add new components and refine existing ones. Please refer to the [jira board][jira] to see what we are currently working on and what we have planned for the future.
+
+[jira]: https://switchcomm.atlassian.net/secure/RapidBoard.jspa?rapidView=246
```

Comment:
This board "Component Library" link is outdated?

```suggestion
[jira]: https://dialpad.atlassian.net/browse/DT
```

Reply:
great find thanks!